rts/Game/Camera.h

```diff
@@ -94,12 +94,8 @@ class CCamera {
 	};
 	struct FrustumLine {
 		int sign = 0;
-
 		float base = 0.0f;
 		float dir = 0.0f;
-
-		float minz = 0.0f;
```

Comment:
Was already unused, only relevant as an alternative to minimap frustum drawing (we will migrate it to something else anyway).